1.Bash-Scripts/website_connectivity

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Test the connectivity to a list of websites
+
+websites=(
+  "www.youtube.com"
+  "www.instagram.com"
+  "www.somesitesdfsdf.com"
+  "www.meta.com"
+)
+
+test_connection(){
+if curl --silent --head --fail "$website" > /dev/null;then
+    echo "$website is reachable"
+  else
+    echo "$website is not reachable"
+  fi
+}
+
+for website in "${websites[@]}"
+do
+  
+  test_connection $website
```

Comment:
Here you are also passing the `website` variable as parameter but not referencing that parameter inside function. Again using global `website` variable inside function.

Check previous comment for details.